tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/QuietBackgroundProcessesPage.xaml

```diff
@@ -25,9 +27,18 @@
                             Margin="{StaticResource MediumLeftRightMargin}"
                             VerticalAlignment="Center"
                             TextTrimming="CharacterEllipsis" />
-                        <StackPanel Orientation="Horizontal" HorizontalAlignment="Right">
-                            <ToggleSwitch IsOn="{x:Bind ViewModel.IsToggleOn, Mode=TwoWay}" IsEnabled="{x:Bind ViewModel.IsToggleEnabled, Mode=OneWay}" />
-                        </StackPanel>
+
+                        <!-- Start/Stop button -->
+                        <ToggleButton
+                            Content="{x:Bind ViewModel.QuietButtonText, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                            IsEnabled="{x:Bind ViewModel.IsFeaturePresent, Mode=OneWay}"
+                            IsChecked="{x:Bind ViewModel.QuietButtonChecked, Mode=OneWay}">
+                            <i:Interaction.Behaviors>
+                                <ic:EventTriggerBehavior EventName="Click">
+                                    <ic:InvokeCommandAction Command="{x:Bind ViewModel.QuietButtonClickedCommand}"/>
+                                </ic:EventTriggerBehavior>
+                            </i:Interaction.Behaviors>
+                        </ToggleButton>
```

Comment:
I don't know off the top of my head if you can set the Command on a ToggleButton directly, but I think you might be able to, in which case this can all become
```suggestion
                            IsChecked="{x:Bind ViewModel.QuietButtonChecked, Mode=OneWay}"
                            Command="{x:Bind ViewModel.QuietButtonClickedCommand}" />
```

Reply:
works :)

I swear I tried that before and it failed.

Reply:
Oh I had been trying,

Click="{x:Bind ViewModel.QuietButtonClickedCommand}"